normandy/selfrepair/static/js/self_repair.js

```diff
@@ -0,0 +1,22 @@
+import Normandy from './normandy_driver.js';
+import {fetchRecipes, filterContext, doesRecipeMatch, runRecipe} from './self_repair_runner.js';
+
+// Actually fetch and run the recipes.
+fetchRecipes().then(recipes => {
+  let context = filterContext().then(context => {
```

Comment:
I did that because JEXL accepts Promises as context values and automatically waits for them to resolve, but your point below about country is still an issue. Nice catch!